internal/notifier/github.go

```diff
@@ -87,6 +88,66 @@ func NewGitHub(addr string, token string, certPool *x509.CertPool) (*GitHub, err
 	}, nil
 }
 
+func NewGitHubApp(addr string, appID int64, installationID int64, privateKey []byte, certPool *x509.CertPool) (*GitHub, error) {
```

Comment:
This func has 0 test coverage. Please add appropriate unit tests. The tests for `NewGitHub` might serve as an example.